### ISSUE TITLE
Joomla 3.5.1 / CiviCRM 4.7.8 Not Syncing

### ISSUE THREAD

**Issue by laybergerF** [lcdservices/CiviCRM-Group-Sync#8]
I have the plugin and component installed and set to sync Joomla Registered users with a CiviCRM group (All Registered Contacts) but when a contact is added to the CiviCRM group, they are not added to the Joomla Registered users group. When I add a user to Joomla, i get the following error, 'CiviCRM 4.3.4 or better required' even thought CiviCRM is version 4.7.8.

**Comment by marchelo2212** [comment]
What was the first version that installed CiviCRM .. ??
When upgrading you made the upgrade to the database that appears just after updating with a message.

**Comment by laybergerF** [comment]
I've been using CiviCRM for several years now so I've upgraded multiple times. I noticed this issues a few months ago, but have not needed it since my site hasn't had a significant number of new users registering,,, now there are several, and I need their username (requested via the CiviCRM registration form) so that I can manually process them for a special Joomla usergroup/task. Yes, I've updated the database each time I've updated CiviCRM. I've looked at the com-users table, and the username isn't saved when the other data collected via my CiviCRM registration page is saved.

**Comment by lcdservices** [comment]
I'll try to carve out time later this week to look into this issue. My workload has been pretty full lately, so I've been limited in what I can do.

**Comment by onderzoekspraktijk** [comment]
I upgraded a site from civicrm 4.6.* to 4.7.10 on a Joomla 3.6.2 site and tested the workings of the group-sync plugin. It works okay for me.
I heard that in the upgrade from Joomla 3.5.* to 3.6.* several changes in the user acl coding were realised. I asked because changes in the the civicrm acl started working again in Joomla 3.6.* , after being broken for ages before.

**Comment by lcdservices** [comment]
that's correct, there was a change (bug) in Joomla 3.5 that prevented you from saving CiviCRM permissions which has since been fixed in 3.6. that may have been the cause of the issue originally described here. would be good to know if the others that reported problems are now seeing it work in Joomla 3.6

**Comment by laybergerF** [comment]
Synchronization now appears to be working on my site, and civiCRM related changes to the ACL are being saved.
